backend/corpora/gale/gale.py

```diff
@@ -0,0 +1,309 @@
+
+            data_location = fix_path_sep(data_location)
+            image_path = fix_path_sep(image_location)
+
+            if issue_date.startswith('[') and issue_date.endswith(']'):
+                issue_date = issue_date[1:-1]
+
+            if issue_date == 'Date Unknown':
+                date_full = None
+            else:
+                date_full = datetime.strptime(issue_date, '%B %d, %Y').strftime('%Y-%m-%d')
+
+            issue_id = filename.split("_")[0]
+            external_file = os.path.join(self.data_directory, data_location, filename)
+            xml_file = os.path.join(self.data_directory, data_location, issue_id + '_Text.xml')
+            if not os.path.isfile(xml_file):
+                print("File {} not found".format(xml_file))
+                continue
+
+            meta = dict(title=publication_tile, image_path=image_path, issue_id=issue_id, external_file=external_file,
+                        date_full=date_full)
+            yield xml_file, meta
+
+    def process_scan(self, src):
+        dst = src
+        base, ext = os.path.splitext(src)
+        if ext.lower() == '.tif':
+            dst = base + '.png'
+            if not os.path.isfile(dst):
+                PIL.Image.open(src).save(dst)
+        return dst
+
+    @property
+    def date(cls):
+        return FieldDefinition(
+            name="date",
+            display_name="Formatted Date",
+            description="Publication date, formatted from the full date",
+            es_mapping={"type": "date", "format": "yyyy-MM-dd"},
+            histogram=True,
+            search_filter=filters.DateFilter(
+                cls.min_date,
+                cls.max_date,
+                description=(
+                    "Accept only articles with publication date in this range."
+                ),
+            ),
+            extractor=extract.Metadata("date"),
+            csv_core=True,
+            visualizations=["resultscount", "termfrequency"],
+        )
+    date_pub = FieldDefinition(
+        name="date_pub",
+        display_name="Publication Date",
+        description="Publication date as full string, as found in source file",
+        es_mapping=keyword_mapping(),
+        results_overview=True,
+        extractor=extract.Metadata("date_full"),
+    )
+    id = FieldDefinition(
+        name="id",
+        display_name="ID",
+        description="Unique identifier of the entry.",
+        es_mapping=keyword_mapping(),
+        extractor=extract.XML(attribute="id"),
+    )
+    issue = FieldDefinition(
+        name="issue",
+        display_name="Issue number",
+        description="Source issue number.",
+        es_mapping=keyword_mapping(),
+        results_overview=False,
+        extractor=extract.Metadata("issue_id"),
+        csv_core=False,
+    )
+    periodical = FieldDefinition(
+        name="periodical",
+        display_name="Periodical name",
+        histogram=True,
```

Comment:
You can leave out `histogram` here, it's not an argument for `FieldDefinition`. (The Periodicals corpus includes it; I guess it was an argument at some point.)